extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -277,6 +277,37 @@
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" fill="none"><path stroke="#fff" stroke-linecap="round" stroke-width="1.9" d="M28.47 17.14c1.15.2 5.46 1.98 5.46 6.92 0 4.95-4.39 6.63-5.46 6.83"/><path stroke="#fff" stroke-linecap="round" stroke-width="1.8" d="M28.47 21.24c.37.08 1.75.79 1.75 2.78 0 1.99-1.4 2.67-1.75 2.74"/><path stroke="#fff" stroke-linecap="round" stroke-width="1.9" d="m18.36 27.48 4.88 4.9a.65.65 0 0 0 1.1-.46V16.09c0-.58-.7-.87-1.1-.45l-4.88 4.94h-2.59c-.94 0-1.7.77-1.7 1.7v3.45c0 .93.75 1.69 1.67 1.7l2.62.05Z"/></svg>') !important;
 }
 
+.i-amphtml-story-mute-audio-control::before {
+  content: "";
+  position: absolute;
+  width: 42px;
+  height: 42px;
+  border-radius: 50%;
+  background-color: #b1b1b1;
+  opacity: 0;
+  /* centered position */
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: auto;
+  /* expand to full circle and then fade out */
+  animation: expand-fade 10s ease-in-out forwards;
+}
+
+@keyframes expand-fade {
+  0%, 40% {
+    transform: scale(0);
+  }
+  50%, 90% {
+    transform: scale(1);
+    opacity: 0.5;
+  }
+  100% {
+    opacity: 0;
+  }
+}
```

Comment:
@processprocess to check whether the animation can be improved. I have one here that's a bit nicer: https://codepen.io/mszylkowski/pen/rNzgxrN

```css
@keyframes pulse-anim {
  0% {
    transform: scale(0.3);
    opacity: 0;
  }
  50% {
    transform: scale(0.9);
    opacity: 0.5;
  }
  100% {
    transform: scale(1);
    opacity: 0;
  }
}
```

Reply:
Great examples!
The `diagonal` animation is good since it expresses crossing out the icon.  
The duration of .5 is okay since it gives time to be visible if the user taps and moves their finger.
We can use this variable for the easing-curve: `--i-amphtml-interactive-ease-out-curve`.

https://github.com/ampproject/amphtml/assets/3860311/24b57721-f4a0-494e-a1fa-42b286d693c8

Reply:
Yeah. I was specifically referencing the `pulse` animation because this PR has a similar effect, and I like the animation curves/keyframes from my demo a bit more than the animation in this PR. 

![sound-toggle](https://github.com/ampproject/amphtml/assets/22420856/2f4bfed9-288c-4782-9586-9ac95fb6d542)

I wasn't considering the other animations but we could ask UX if they have a preference for any of those as well.

Reply:
Got it. I don't think we should add the pulse for a single button. 
Pulse is usually applied to all buttons in a design system. 

As I recall, the design system of Web Stories is independent of Material, which is why all buttons don't have the pulse.

Reply:
Thanks for the suggestions! The UX detail here is not strictly pulse. It actually scales up a bit and waits for 4 seconds and then just fade it out.

I don't have strong preferences about which one we should choose from any of 1) current animation 2) Matias's pulse 3) Philip's diagonal. Let me include our UX expert in the discussion to find out.

Reply:
My Codepen has all those animations and more. It should be easy to implement from it, in case we need it.

Reply:
@ychsieh sorry for the misleading comment earlier in our chat.  Spent more time reading the comments above and got my memory refreshed. I agree what Philip said about the ripple. The only thing is I feel 0.5s is not enough to see  the diagonal animation while moving finger away from the button.

Reply:
I think we are talking about two different scenarios here: 1) the user clicks unmute button 2) the user clicks audio sticker. The animation for 1) should be the diagonal or the pulse one and requires shorter duration so that users can see it immediately after they click it. On the other hand, the animation for 2) is more for reminding people that they can toggle it again on the system header after the audio sticker disappears. 

We can make the animation of clicking unmute button better but I think it is out of scope for audio sticker changes here. What do you guys think?

Reply:
Done. Could you guys take another look? Thanks!

@mszylkowski @processprocess